### ISSUE TITLE
UnityExport folder is sometimes not there in the xCode

### ISSUE THREAD

**Issue by im-aditya** [f111fei/react-native-unity-view#107]
Hi,

This does not happen all the time but when I export my iOS project from Unity using **Build > Export iOS**, sometimes the _UnityExport_ folder (which appears as a yellow folder in xCode) is not there in the Project navigator. In this state if I try to compile the project then I get linker errors. 

Is there any other way I can add this folder to xCode or is there any way I can check what is wrong ?

Thanks

**Comment by JanOwiesniak** [comment]
I stumbled across this issue several times. What works for me (most of the time) is the following routine:

1. Delete UnityExport from XCode
2. Clean project from XCode
3. Close XCode
4. Export UnityExport from Unity
5. Open XCode

**Comment by im-aditya** [comment]
There has to be some configuration or project file where this information is getting stored. If we can simply check whether that file or configuration is properly set or not ??

**Comment by JanOwiesniak** [comment]
Agree. My approach is not fixing the actual issue, instead i'm fighting the symptoms of something deeper. Neverless this might be a good hotfix for some folks right now. I'm not even sure that this issue has something to do with react-native-unity-view. Could be a XCode caching issue as well because cleaning the XCode project folder often fixes this issue for me.
